3rd-party/CMakeLists.txt

```diff
@@ -1 +1 @@
-add_subdirectory(spdlog)
\ No newline at end of file
+add_subdirectory(spdlog)
```

Comment:
Add to this file:
```cmake
add_subdirectory(libpqxx)
```

And in the base cmakelists.txt file, link alicia-libserver against the `libpqxx::libpqxx` target